### PR TITLE
Added 14 Google App Engine samples.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

**Comment by jonparrott** [comment]
This looks good to me.

 What ended up being the major differences between the sendgrid sample?

**Comment by jmdobry** [comment]
I'm not done yet. I will assign to you when it's ready for review.

**Comment by jmdobry** [comment]
Ready for review

File: appengine/websockets/views/index.jade [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    div
+      p Status:
+      pre(id="echo-status")
+    //- [END form]
+
+    script(src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js")
+    script.
+      // [START js]
+      $(function() {
+        /* The external ip is determined by main.py and passed into the template. */
```
Comment (jonparrott) [inline appengine/websockets/views/index.jade]: This is my fault, but it says 'main.py' here. ;)

File: appengine/analytics/package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache Version 2.0",
+  "author": "Google Inc.",
+  "engines": {
+    "node": "~4.2"
+  },
+  "scripts": {
+    "start": "node app.js",
+    "monitor": "nodemon app.js",
```
Comment (JustinBeckwith) [inline appengine/analytics/package.json]: nit - should we have nodemon as a devDependency?

File: appengine/bower/package.json [hunk 3]
```diff
@@ -4,13 +4,14 @@
...
   "license": "Apache Version 2.0",
+  "author": "Google Inc.",
   "engines": {
     "node": "~4.2"
   },
   "scripts": {
     "start": "node server.js",
     "postinstall": "bower install --config.interactive=false",
-    "deploy": "gcloud preview app deploy app.yaml"
+    "deploy": "gcloud preview app deploy"
```
Comment (JustinBeckwith) [inline appengine/bower/package.json]: :smile:

File: appengine/express-memcached-session/README.md [hunk 4]
```diff
@@ -48,44 +40,36 @@ In your terminal window, press Ctrl+C to exit the web server.
...
-runtime: nodejs
-vm: true
-env_variables:
-  PORT: 8080
-  MEMCACHE_URL: memcache:11211
-```
+    runtime: nodejs
+    vm: true
+    env_variables:
+      PORT: 8080
```
Comment (JustinBeckwith) [inline appengine/express-memcached-session/README.md]: I don't think we need to do this anymore

File: appengine/express/views/layout.jade [hunk 5]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2015, Google, Inc.
+// Copyright 2016, Google, Inc.
```
Comment (JustinBeckwith) [inline appengine/express/views/layout.jade]: Not that this really matters but....  the way Copyrights are supposed to work, is that you include the range of years from the date it originally was released.  

  "the year of first publication of the work; in the case of compilations or derivative works incorporating previously published material, the year date of first publication of the compilation or derivative work is sufficient"

**Comment by jsimonweb** [comment]
LGTM

**Pull request merged**